tests/utils.ts

```
@@ -90,10 +93,28 @@ export function u32ArrayToHex(
 
 export function generateOrderId(
   chainId: string,
-  secretHash: number[],
-  intiatorAddress: string
+  initiatorAddress: string,
+  redeemerAddress: string,
+  timelock: BigInt,
```

Biome (1.9.4)

[error] Don't use 'BigInt' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'bigint' instead


(lint/complexity/noBannedTypes)